.github/workflows/publish.yaml

```diff
@@ -41,9 +41,9 @@ jobs:
           git config user.email "${{ github.actor}}@users.noreply.github.com"
 
           if [ ${{ github.ref_name }} = development ]; then
-            npx lerna publish prerelease --conventional-commits --conventional-prerelease --preid dev-$(date +%Y%m%dT%H%M%S) --no-changelog --no-git-tag-version --dist-tag dev --no-push --yes
+            npx lerna publish prerelease --exact --conventional-commits --conventional-prerelease --preid dev-$(date +%Y%m%dT%H%M%S) --no-changelog --no-git-tag-version --dist-tag dev --no-push --yes
           elif [ ${{ github.ref_name }} = next ]; then
-            npx lerna publish prerelease --conventional-commits --conventional-prerelease --preid alpha --yes --no-private --dist-tag latest
+            npx lerna publish prerelease --exact --conventional-commits --conventional-prerelease --preid alpha --yes --no-private --dist-tag latest
           else
             npx lerna publish --conventional-commits --conventional-graduate --yes --no-private --create-release github --dist-tag latest
```

Comment:
```suggestion
            npx lerna publish --exact --conventional-commits --conventional-graduate --yes --no-private --create-release github --dist-tag latest
```

Reply:
I think you'd want to do it on GA releases too, right?

Reply:
I wasn't sure about that as exact versions in public libraries are generally not a good thing. I thought that at least for the dev/ alpha version is crucial for those to exactly specify all related deps. On the official release, I would try to relying on version ranges.
As described in the PR description, the original challenge was that it resolved `^0.0.2-alpha.15` to `0.0.2-dev-20240114T223125.0`. This means that even the latest alpha branch is eventually "lower" than any dev branch and thus leads to very outdated references. On the GA releases, I wouldn't expect something like this to happen. Not sure though, so let's keep an eye out on this and add it later if this really the case.